### PR TITLE
fix(Onboarding): align the 'Before you get started' popup with design

### LINKED ISSUE #6020: Popup 'Before you get started' differs from the designs
# Bug Report

## Description
The size of the text is smaller than it's shown on the designs

## Steps to reproduce
1. Install the app
2. Go to create a new account
3. Compare the popup 'Before you get started'  with the designs (https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1023%3A298171)

#### Expected behavior
The text has the same size as it's shown on the designs
image

#### Actual behavior
The size of the text is smaller than it's shown on the designs
image

### Additional Information

Status desktop version: https://ci.status.im/job/status-desktop/job/branches/job/macos/job/master/2163/
Operating System: macOS Monterey 12.3 Beta (21E5227a)

### PR DESCRIPTION
**Pull request by caybro**
correct sizing, fonts, spacings, margins, etc

Fixes #6020

Needs status-im/StatusQ#776

### What does the PR do

Aligns the 'Before you get started' popup with design

### Affected areas

Onboarding/BeforeGetStarted

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

Actual app:
Snímek obrazovky z 2022-07-14 14-29-17

Design:
Snímek obrazovky z 2022-07-14 15-17-22

### CODE REVIEW

File: ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml [hunk 1]
```diff
@@ -12,47 +12,52 @@ import StatusQ.Popups 0.1
 StatusModal {
```
Comment (osmaczko) [inline ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml:12]: Could you please use `StatusDialog` as the root component? `StatusModal` is deprecated.

**Review by osmaczko — COMMENTED** [review]
Could you please also link StatusQ PR?

File: ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml (hunk 1 above)
Comment (caybro) [reply ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml]: @osmaczko done, also linked the corresponding StatusQ PR

File: ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml [hunk 2]
```diff
@@ -1,58 +1,77 @@
...
 import StatusQ.Controls 0.1
-import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
 
-StatusModal {
+StatusDialog {
     id: popup
 
+    width: 480
+    height: 318
```
[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml]: Done

**Review by osmaczko — APPROVED** [review]
Thanks! Please squash commits before merge.

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Pull request merged**